pandas/tests/extension/decimal/array.py

```diff
@@ -59,14 +59,28 @@ def __init__(self, values, dtype=None, copy=False, context=None):
         values = np.asarray(values, dtype=object)
 
         self._data = values
-        # Some aliases for common attribute names to ensure pandas supports
-        # these
-        self._items = self.data = self._data
-        # those aliases are currently not working due to assumptions
-        # in internal code (GH-20735)
-        # self._values = self.values = self.data
         self._dtype = DecimalDtype(context)
 
+    # aliases for common attribute names, to ensure pandas supports these
+    @property
+    def _items(self):
+        return self._data
+
+    @property
+    def data(self):
+        return self._data
+
+    # those aliases are currently not working due to assumptions
```

Comment:
don't added commented code unless its germane (its not here)

Reply:
Which assertion do you mean?

Reply:
There were 2 commented out attributes. I tuned them into properties with the rest, then commented them out to not delete what the author saw fit to leave in. Not sure what to do here.

Reply:
Yes, it is fine to keep this in comments. 

But for briefness, I think it is enough to have only one property (I would take `data`) and then you can simply do `items = data` as alias, and keep the `values` and `_values` in a similar way as alias in comments.